### DISCUSSION TITLE
Linux mint?

### DISCUSSION THREAD

**Discussion by JetpackJackson** [dnschneid/crouton#4564] [Q&A] [answered]
Will linux mint be supported in crouton?

**Comment by DennisLfromGA** [comment, marked as answer]
@JetpackJackson,

I doubt it unless someone opens a pull request for it and it gets reviewed and merged.

crouton used to have a Cinnamon target but it has been dead for a while now - #832 (comment)
Quite a while ago there was some discussion of bringing it back, and even mention of mint - #1000

*Hope this helps,*
-**DennisLfromGA**

**Reply by JetpackJackson** [reply]
Ok, thank you for letting me know!
